config.env

```diff
@@ -2,7 +2,7 @@ BN_WS_IP=
 BN_WS_PUBLIC_KEY=
 BN_WS_PRIVATE_KEY=
 BN_WS_ETH_PRIVATE_KEY=
-BN_SP_UPDATER_INFURA_URL=
+BN_UPDATER_SP_INFURA_URL=
```

Comment:
It's better to have all variables starting with `BN_UPDATER` beside each other.

Reply:
It's done
https://github.com/BrightID/BrightID-Node/commit/c79d3aca89f882dd942ae894552b87b438bb9e68